app/src/test/java/ingsw/group1/findmyphone/cryptography/RandomStringGeneratorTest.java

```diff
@@ -0,0 +1,102 @@
+package ingsw.group1.findmyphone.cryptography;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertTrue;
+
+import java.util.ArrayList;
+import java.util.Collection;
+
+import ingsw.group1.msglibrary.SMSMessage;
+import ingsw.group1.msglibrary.SMSPeer;
+
+/**
+ * Tests the encryption and decryption with generating random text to cypher / decipher with a random generated password.
+ *
+ * @author Pardeep
+ */
+@RunWith(Parameterized.class)
+public class RandomStringGeneratorTest {
+    private final int MINIMUM_STRING_LENGTH = 1;
+    private final int MAXIMUM_STRING_LENGTH = 60;
+
+    private static final int NUM_REPEATS = 1000;
+
+    @Parameterized.Parameters()
+    public static Collection<Object[]> data() {
+        Collection<Object[]> out = new ArrayList<>();
+        for (int i = 0; i < NUM_REPEATS; i++) {
+            out.add(new Object[0]);
+        }
+        return out;
+    }
+
+    @Test
+    public void RandomIntegerGeneratorTest() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The JUnit 4 test method name 'RandomIntegerGeneratorTest' doesn't match '[a-z][a-zA-Z0-9]*'](https://app.codacy.com/manual/RicDeZen/ingsw-group1-findmyphone/pullRequest?prid=4958670)

Reply:
Means the test name should start (as all methods usually do) with a lower case letter